tests/Tests/400__ods_dashboard/adminUITest.robot

```diff
@@ -0,0 +1,159 @@
+*** Settings ***
+Library         OpenShiftCLI
+Library         OpenShiftLibrary
+Resource        ../../Resources/Common.robot
+Resource        ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+
+
+*** Test Cases ***
+Verify If Unauthorisez User Can Not Change The Permission
+    [Tags]  ODS-1661
+    ...     ODS-1660
+    Set Library Search Order  SeleniumLibrary
+    Check User Management Option Is Available For The User   ${TEST_USER.USERNAME}   ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+    Clear All The Input Values In User Managermant
```

Comment:
typo in `Management`

Reply:
the kw name could be more self explanatory maybe. Try sth like `Clear User Management Settings`